moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -1013,7 +1019,7 @@ void MotionPlanningDisplay::changePlanningGroup(const std::string& group)
     planning_group_property_->setStdString(group);
   }
   else
-    ROS_ERROR("Group [%s] not found in the robot model.", group.c_str());
+    RCLCPP_ERROR("Group [%s] not found in the robot model.", group.c_str());
```

Comment:
You need a Logger name here, you have to declare it as 
``rclcpp::Logger LOGGER_MOTION_PLANNING_RVIZ_PLUGIN = rclcpp::get_logger("motion_planning_rviz_plugin").get_child("motion_planning_display");`` for example, or if it is already declared a node, you can just use it. 
```suggestion
    RCLCPP_ERROR(node_->get_logger(),"Group [%s] not found in the robot model.", group.c_str());
```
or
```suggestion
    RCLCPP_ERROR(LOGGER_MOTION_PLANNING_RVIZ_PLUGIN,"Group [%s] not found in the robot model.", group.c_str());
```